README.md

```diff
@@ -37,14 +38,22 @@ pod 'XcodeServerSDK'
 
 Create the server config object with the server's URL, username and password.
 ```swift
-let config = XcodeServerConfig(host: "https://127.0.0.1", user: "IRuleBots", password: "superSecr3t")
-let server = XcodeServerFactory.server(config)
+do {
+    let config = XcodeServerConfig(host: "https://127.0.0.1", user: "IRuleBots", password: "superSecr3t")
+} catch {
+    fatalError("Something went wrong!")
+}
+```
+
+Instantiate `XcodeServer`.
+```swift
+let server = XcodeServer(config: config, endpoints: endpoints)
```

Comment:
Where? I can't find it here: https://github.com/czechboy0/XcodeServerSDK/blob/master/XcodeServerSDK/XcodeServer.swift